genesis/options/textures.py

```diff
@@ -26,28 +26,41 @@ def __init__(self, **data):
     def check_dim(self, dim):
         raise NotImplementedError
 
+    def check_simplify(self):
+        raise NotImplementedError
+
     def apply_cutoff(self, cutoff):
-        pass
+        raise NotImplementedError
 
 
 class ColorTexture(Texture):
     """
-    A texture with a single color.
+    A texture that consists of a single color.
 
     Parameters
     ----------
-    color : tuple, shape (3,)
-        RGB color value.
+    color : list of float
+        A list of color values, stored as tuple, supporting any number of channels within the range [0.0, 1.0]. Default is (1.0, 1.0, 1.0).
     """
 
-    color: tuple = (1.0, 1.0, 1.0)
+    color: Union[float, List[float]] = (1.0, 1.0, 1.0)
```

Comment:
What is supposed to be a "float" color? Besides, do not use `List[float]` but `typing.Sequence[float]`.